crates/taplo-wasm/src/lsp.rs

```diff
@@ -83,3 +83,5 @@ impl Sink<rpc::Message> for WasmLspInterface {
         std::task::Poll::Ready(Ok(()))
     }
 }
+
+unsafe impl Send for WasmLspInterface {}
```

Comment:
Why is this safe? `js_sys::Function` doesn't seem to implement `Send`.

Reply:
Oh actually this isn't safe, I made a mistake thinking the writer also required to implement Send which is not the case. The `Sink` MessageWriter type doesn't require Send so it might be better not to have `Arc<Mutex<>>` lock instead replace with more performant Rc and fix clippy suggestions?